metrics/network/network-metrics-iperf3.sh

```diff
@@ -16,46 +16,41 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
-# Test inter-container network bandwidth and jitter using iperf3
+# This test will measure the following network essentials:
+# - bandwith simplex
+# - bandwith duplex
+# - jitter
+#
+# These metrics will be got from the interconnection between a container
+# server and client using the iperf tool.
+# container-server <----> container-client
 
 set -e
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
-
-source "${SCRIPT_PATH}/lib/network-test-common.bash"
+source "${SCRIPT_PATH}/lib/network-common.bash"
+source "${SCRIPT_PATH}/../lib/common.bash"
 
 # Port number where the server will run
-port=5201:5201
+port="5201:5201"
 # Image name
-image=gabyct/network
+image="gabyct/network"
 # Measurement time (seconds)
-time=5
-# Name of the containers
-server_name="network-server"
-client_name="network-client"
-
-# Arguments to run the client
-if [ "$RUNTIME" == "runc" ];then
-	extra_args="-ti --privileged --rm"
-else
-	extra_args="-ti --rm"
-fi
-
-## Test name for reporting purposes
-test_name="network metrics iperf3"
-
-# This script will perform all the measurements using a local setup using iperf3
+transmit_time=5
```

Comment:
I'd probably be happier if we s/time/count/  `time` always makes me think it is an 'amount of time' measure rather than a 'number of times' one.

Reply:
yeah, that sounds better

Reply:
checking tool options it seems that it is a unit time `-t, --time      #        time in seconds to transmit for (default 10 secs)`